vhdbuilder/packer/install-dependencies.sh

```diff
@@ -413,29 +413,20 @@ done
 # v1.16.13-hotfix.20200714.1
 # v1.17.7-hotfix.20200714.1
 # v1.17.9-hotfix.20200714.1
-# v1.18.4-hotfix.20200624.1
-# v1.18.6
+# v1.18.4-hotfix.20200626.1
+# v1.18.6-hotfix.20200723.1	
 K8S_VERSIONS="
-1.14.7-hotfix.20200408.1
-1.14.8-hotfix.20200529.1
-1.15.7-hotfix.20200326
-1.15.10-hotfix.20200408.1
-1.15.11-hotfix.20200714.1
-1.15.12-hotfix.20200714.1
-1.16.7-hotfix.20200601.1
-1.16.8.1
-1.16.9-hotfix.20200529.1
-1.16.10-hotfix.20200714.1
-1.16.13-hotfix.20200714.1
-1.17.3-hotfix.20200601.1
-1.17.4.1
-1.17.5.1
-1.17.7-hotfix.20200714.1
-1.17.9-hotfix.20200714.1
-1.18.1.1
-1.18.2-hotfix.20200624
-1.18.4-hotfix.20200624.1
-1.18.6
+v1.15.11-hotfix.20200714.1
+v1.15.12-hotfix.20200714.1
+v1.16.9-hotfix.20200529.1	
+v1.16.10-hotfix.20200714.1
+v1.16.13-hotfix.20200714.1
+v1.17.3-hotfix.20200601.1
+v1.17.7-hotfix.20200714.1
+v1.17.9-hotfix.20200714.1
+v1.18.2-hotfix.20200624.1
```

Comment:
why did thi magically get a .1?  at the end

Reply:
I think that's the build number. It is built by upstream

Reply:
It can also be .2 .3 .4. It just means the 2nd, 3rd, 4th build of the day I think.

This is an example where it can be .2

https://dev.azure.com/AzureContainerUpstream/Kubernetes/_build/results?buildId=8958&view=results